centreon/www/front_src/src/ApiTokens/api/models.ts

```diff
@@ -10,3 +10,8 @@ export interface DeleteTokenEndpoint {
   tokenName: string;
   userId: number;
 }
+
+export interface PatchTokenEndpoint {
```

Comment:
this is a duplication of the interface DeleteTokenEndpoint

Reply:
Done